src/frontend/typer/behandling.ts

```diff
@@ -47,6 +53,13 @@ export const behandlingÅrsak: Record<BehandlingÅrsak, string> = {
     OMREGNING_6ÅR: 'Omregning 6 år',
     OMREGNING_18ÅR: 'Omregning 18 år',
     MIGRERING: 'Migrering',
+    /** De neste er revurderingsårsaker for tilbakekrevingsbehandlinger **/
+    REVURDERING_KLAGE_NFP: 'Revurdering etter klage',
+    REVURDERING_KLAGE_KA: 'Revurdering etter klage',
+    REVURDERING_OPPLYSNINGER_OM_VILKÅR: 'Nye opplysninger',
+    REVURDERING_OPPLYSNINGER_OM_FORELDELSE: 'Nye opplysninger',
```

Comment:
supert at dette blir mappet til lik årsak som vises for de andre behandlingene! Jeg lurer også litt på om man bør vise "Ikke fastsatt" som "Ikke vurdert" (eller omvendt) for å være konsistent, eller blir ikke det helt riktig kanskje? :)